get-dependencies.js

```diff
@@ -28,9 +28,7 @@ const getDirectDependencies = function (modulePath, options) {
 					if (options.ignoreExternal) return null;
 					if (builtinModules.has(depPath.split("/")[0])) return null;
 				}
-				return cjsResolve(
-					dir, depPath
-				)(pathData => {
+				return cjsResolve(dir, depPath, { silent: options.ignoreMissing })(pathData => {
```

Comment:
I believe right way to fix it is to pass `silent: true`, 

As below logic assumes that if module is not found `null` would be returned (and without `ignoreMissing` a meaningful error would be returned: https://github.com/medikoo/ncjsm/blob/b45070ec1341223670e5ed1a106a8686664ea235/get-dependencies.js#L36)